molecule/config.py

```diff
@@ -337,6 +337,8 @@ def _interpolate(self, stream, env, keep_string):
             util.sysexit_with_message(msg)
 
     def _get_defaults(self):
+        scenario_name = \
```

Comment:
Please use brackets instead of EOL escaping
```suggestion
        scenario_name = (
```